graphql/src/adapters/graphql/initialized_adapters/__init__.py

```diff
@@ -0,0 +1,3 @@
+from src.adapters.word_generation.word_generation import UUIDWordGeneration
+
```

Comment:
This `initialized_adapters` folder confuses me a bit. Wouldn't this happen in resolvers since it is there we are applying the adapter to the use case? In the future would you instantiate all the adapters here first before resolvers calls them?

Reply:
I was attempting to replicate https://git.earthdata.nasa.gov/projects/ORCA/repos/orca-prototype-graphql/browse/src/server/adapters/api/graphql/storage/__init__.py?at=refs%2Fheads%2Ffeature%2Ffastapi-strawberry-clean
but with changes that allow us to more easily point to a different storage adapter if needed.
This file also forces us to always have a set map from each interface to its implementing adapter.